tests/core/pyspec/eth2spec/test/lightclient_patch/epoch_processing/test_process_final_updates.py

```diff
@@ -0,0 +1,37 @@
+from eth2spec.test.context import (
+    PHASE0, PHASE1,
+    with_all_phases_except,
+    spec_state_test,
+)
+from eth2spec.test.helpers.state import transition_to
+from eth2spec.test.phase0.epoch_processing.run_epoch_process_base import (
```

Comment:
do we want to be strict w/ "mixing" forks here?

`run_epoch_process_base` (or the relevant subset) can likely go into some kind of epoch processing helpers